Drakhtar/Dialog.cpp

```diff
@@ -2,29 +2,28 @@
 
 #include "Dialog.h"
 
-Dialog::Dialog(Game *game, ifstream &file, Font *textFont, SDL_Rect dialogRect) : dialogRect_(dialogRect)
+Dialog::Dialog(Game* game, ifstream& file, Font* textFont, SDL_Rect dialogRect, int lineJumpLimit)
+: dialogRect_(dialogRect)
 {
     readFromFile(file);
     characterPortraitSprite = new GameObject(TextureManager::get(spriteText),
-                                             Vector2D<int>(dialogRect_.x + 200, dialogRect_.y + 345),
-                                             Vector2D<int>(dialogRect_.w * 150, dialogRect_.h * 150));
+                                           Vector2D<int>(dialogRect_.x + 200, dialogRect_.y + 345),
+                                           Vector2D<int>(dialogRect_.w * 150, dialogRect_.h * 150));
 
     SDL_Color textColor;
     textColor.r = 0;
     textColor.g = 0;
     textColor.b = 0;
     textColor.a = 1;
 
-    characterNameSprite = new Text(game->getRenderer(),
-                                   textFont,
-                                   Vector2D<int>(dialogRect_.x + 600, dialogRect_.y + 405),
-                                   textColor,
-                                   characterName);
-    dialogTextSprite = new Text(game->getRenderer(),
-                                textFont,
-                                Vector2D<int>(dialogRect_.x + 375, dialogRect_.y + 480),
-                                textColor,
-                                dialogText);
+    characterNameSprite =
```

Comment:
Same as before, it'd look better written the way it was before. Also, isn't this edited by others?

cc: @MiguelZh @Dankgard